frontend/src/app/components/edit-recipe/edit-recipe.component.html

```diff
@@ -15,7 +15,7 @@ <h2>Edit</h2>
       <br/>
          <form [formGroup]="updateRecipeForm" class="create-form">
             <div class="header-info">
-               <input type="text" matInput placeholder="Author" formControlName="author" autocomplete="off">
+               <input type="text" matInput placeholder="Author" formControlName="author" autocomplete="off" readonly>
```

Comment:
Small bug I noticed, we don't want the author field to be editable when editing a recipe. This may not be the best fix but it does the job for now